azurelinuxagent/common/osutil/default.py

```diff
@@ -287,12 +287,12 @@ def get_instance_id(self):
         '''
         if os.path.isfile(PRODUCT_ID_FILE):
             s = fileutil.read_file(PRODUCT_ID_FILE).strip()
-
+            
         else:
             rc, s = shellutil.run_get_output(DMIDECODE_CMD)
             if rc != 0 or UUID_PATTERN.match(s) is None:
                 return ""
-
+              
```

Comment:
NIT: whitespace

Reply:
Accidentally removed whitespace line 295.  Performed a commit to put it back.  This now matches the original code.